net/nqos/files/nqos.config

```diff
@@ -0,0 +1,17 @@
+#
```

Comment:
Default config should have at least some default rules uncommented. Right now the example may be a bit confusing, as it is not obvious which options are mandatory. 

And the example does not say what is the default for optional "unit")

It would also be great to have an "enabled" switch 0/1 in the config file, and defaulting to 0, so that the package can be installed in the image and still be disabled by default.